src/modules/sensors/vehicle_magnetometer/VehicleMagnetometer.cpp

```diff
@@ -209,6 +209,15 @@ void VehicleMagnetometer::UpdateMagBiasEstimate()
 		if (_magnetometer_bias_estimate_sub.copy(&mag_bias_est)) {
 			bool parameters_notify = false;
 
+			bool external_mag_available = false;
+
+			for (unsigned mag_index = 0; mag_index < MAX_SENSOR_COUNT; mag_index++) {
+				if (_calibration[mag_index].external() && _calibration[mag_index].enabled()) {
+					external_mag_available = true;
```

Comment:
Here should we only set `external_mag_available` if there's a corresponding valid magnetometer_bias_estimate?

Reply:
Yes, probably `valid` and `stable` to be sure that the external mag will actually be calibrated.

Reply:
This has been addressed in the mean time, right?

Reply:
yes